src/schema/ecommerce/submit_order_mutation.js

```diff
@@ -19,7 +19,7 @@ const SubmitOrderInputType = new GraphQLInputObjectType({
 
 export const SubmitOrderMutation = mutationWithClientMutationId({
   name: "SubmitOrder",
-  decription: "Submitss an order",
+  description: "Submits an order",
```

Comment:
looks like someone installed the vscode spellchecker ( for others - https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker )